### PR TITLE
add luxaio

### PR DESCRIPTION
**Pull request by FCtar**
Hi,

pls add this new target for a AIO from Lumenier.

KR

### CODE REVIEW

**Review by x4FF3 — APPROVED** [review]
LGTM

**Comment by Asizon** [comment]
Please @x4FF3 , PR's need 24H before merged to can review by other reviewers. Can we revert this merge?

**Pull request merged**
